src/manifold/stream/queue.clj

```diff
@@ -26,10 +29,14 @@
      :source?     true})
 
   (take [this default-val blocking?]
-    (if blocking?
+    (cond
+      (.get is-closed)
+      (d/success-deferred default-val)
```

Comment:
This doesn't match the behavior of sources elsewhere in Manifold. Closing actually means no more puts are allowed, but takes are fine until you run out of data, at which point it returns the default-val (or nil if no default-val).

I just noticed the docstring for close! is wrong, can you remove the part about "can't emit" as part of this?

Reply:
> since that would be more a documentation change would you prefer such in another PR?

Nah, it's just extra overhead.

> Could you elaborate a bit on what you meant by this?

I think I was still learning Manifold as a new maintainer when I wrote that. The close! docstring probably fooled me, too 😉